core/src/test/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolverTests.java

```diff
@@ -641,7 +642,7 @@ public void testConcreteIndicesWildcardAndAliases() {
         // when ignoreAliases option is set, concreteIndexNames resolves the provided expressions
         // only against the defined indices
         IndicesOptions ignoreAliasesOptions = IndicesOptions.fromOptions(false, false, true, false, true, false, true);
-        
+
```

Comment:
unneeded change?

Reply:
should i remove this? my intellij settings might be a bit aggressive :)